monkey/infection_monkey/control_channel.py

```diff
@@ -0,0 +1,52 @@
+import json
+import logging
+
+import requests
+
+from common.common_consts.timeouts import SHORT_REQUEST_TIMEOUT
+from infection_monkey.config import GUID, WormConfiguration
+from infection_monkey.control import ControlClient
+from monkey.infection_monkey.i_control_channel import IControlChannel
+
+requests.packages.urllib3.disable_warnings()
+
+logger = logging.getLogger(__name__)
+
+
+class ControlChannel(IControlChannel):
+    def should_agent_stop(self) -> bool:
+        server = WormConfiguration.current_server
```

Comment:
`server` is set in multiple methods. It's a sign that it should be a class variable.